algebra/src/lib.rs

```diff
@@ -1,28 +1,29 @@
 #![cfg_attr(docsrs, feature(doc_auto_cfg))]
-#![deny(missing_docs)]
```

Comment:
please keep it for every crate, this will force to add comments, which makes the code more readable.